src/cromshell/metadata/command.py

```diff
@@ -0,0 +1,104 @@
+import logging
+
+import click
+import requests
+
+from cromshell.utilities import cromshellconfig, http_utils, io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="metadata")
+@click.argument("workflow_id")
+@click.option(
+    "-nes",
+    "--not_expand_subworkflow",
```

Comment:
I'm not a huge fan of this name...  I would probably go `--dont-expand-subworkflows` instead.

Reply:
done